New package: Bitmutex.Winhider version 0.8.10

aamitn opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/259560)

aamitn committed:
New package: Bitmutex.Winhider version 0.8.10

aamitn commented:
> @aamitn please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement
> ### Contribution License Agreement
> 
> This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> 
>     1. **Definitions**.
>        **“Code”** means the computer software code, whether in human-readable or machine-executable form,
>        that is delivered by You to Microsoft under this Agreement.
>        **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
>        approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
>        **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
>        Project, including but not limited to communication on electronic mailing lists, source code control
>        systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
>        discussing and improving that Project, but excluding communication that is conspicuously marked or
>        otherwise designated in writing by You as “Not a Submission.”
>        **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
>        associated comments and documentation.
> 
>     2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
>        Project. This Agreement covers any and all Submissions that You, now or in the future (except as
>        described in Section 4 below), Submit to any Project.
> 
>     3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
>        Should You wish to Submit materials that are not Your original work, You may Submit them separately
>        to the Project if You (a) retain all copyright and license information that was in the materials as You
>        received them, (b) in the description accompanying Your Submission, include the phrase “Submission
>        containing materials of a third party:” followed by the names of the third party and any licenses or other
>        restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
>        guidelines concerning Submissions.
> 
>     4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
>        for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
>        Submission is made in the course of Your work for an employer or Your employer has intellectual
>        property rights in Your Submission by contract or applicable law, You must secure permission from Your
>        employer to make the Submission before signing this Agreement. In that case, the term “You” in this
>        Agreement will refer to You and the employer collectively. If You change employers in the future and
>        desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
>        and secure permission from the new employer before Submitting those Submissions.
> 
>     5. **Licenses**.
> 
> 
>     * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
>       indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
>       Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
>       the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
>       parties.
> 
>     * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
>       indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
>       Your patent claims that are necessarily infringed by the Submission or the combination of the
>       Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
>       import or otherwise dispose of the Submission alone or with the Project.
> 
>     * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
>       No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
>       granted by implication, exhaustion, estoppel or otherwise.
> 
> 
>     6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
>        licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
>        have disclosed under Section 3). You represent that You have secured permission from Your employer to
>        make the Submission in cases where Your Submission is made in the course of Your work for Your
>        employer or Your employer has intellectual property rights in Your Submission by contract or applicable
>        law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
>        have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
>        You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
>        REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
>        EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
>        PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
>        NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> 
>     7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
>        You later become aware that would make Your representations in this Agreement inaccurate in any
>        respect.
> 
>     8. **Information about Submissions**. You agree that contributions to Projects and information about
>        contributions may be maintained indefinitely and disclosed publicly, including Your name and other
>        information that You submit with Your Submission.
> 
>     9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
>        the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
>        Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
>        exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
>        defenses of lack of personal jurisdiction and forum non-conveniens.
> 
>     10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
>         supersedes any and all prior agreements, understandings or communications, written or oral, between
>         the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

@microsoft-github-policy-service agree company="Bitmutex Technlogies"

stephengillie commented:
Automatic Validation ended with:
> No errors to post.

(Automated response - build 1058.)

ItzLevvie commented on manifests/b/Bitmutex/Winhider/0.8.10/Bitmutex.Winhider.installer.yaml:
```diff
@@ -0,0 +1,12 @@
+
+PackageIdentifier: Bitmutex.Winhider
+PackageVersion: 0.8.10
+InstallerType: inno
```
```suggestion
InstallerType: inno
Scope: machine
ProductCode: "{4896775D-F364-4AF8-AD6C-946EE5F49D95}_is1"
```

ItzLevvie reviewed (changes requested):
Looks like it requires VCRedist dependency to be included with the manifest:
<img width="1238" alt="Screenshot 2025-05-26 at 5 21 48 PM" src="https://github.com/user-attachments/assets/03e52992-b9f8-4b29-b804-497277462147" />

aamitn committed:
Update manifests/b/Bitmutex/Winhider/0.8.10/Bitmutex.Winhider.locale.en-US.yaml

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

aamitn committed:
Update manifests/b/Bitmutex/Winhider/0.8.10/Bitmutex.Winhider.installer.yaml

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

aamitn committed:
Update Bitmutex.Winhider.installer.yaml

added vc redistributable x86 and x64 dependencies

aamitn committed:
Update Bitmutex.Winhider.locale.en-US.yaml

aamitn committed:
Update Bitmutex.Winhider.installer.yaml

aamitn commented:
> Looks like it requires VCRedist dependency to be included with the manifest: <img alt="Screenshot 2025-05-26 at 5 21 48 PM" width="1238" src="https://private-user-images.githubusercontent.com/11600822/447636827-03e52992-b9f8-4b29-b804-497277462147.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDgyNzk3MjEsIm5iZiI6MTc0ODI3OTQyMSwicGF0aCI6Ii8xMTYwMDgyMi80NDc2MzY4MjctMDNlNTI5OTItYjlmOC00YjI5LWI4MDQtNDk3Mjc3NDYyMTQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTI2VDE3MTAyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBiM2QzOTkxZGUwZDA1YmNkMDUwMzUwMDc0YWU2MDlmZTgzYTVmZGNhZmMyOTg3ZTJkYjBlZmFhYWU0MzE3ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.4ohXiaCna3-6x2WR1Yn6u4gfGX6KQz9vtqrAlKPGg5M">

@ItzLevvie 

- Added the required dependencies.
- Reverted back the package `PackageName` to  `winhider` , as there is a preference for all lowercase naming of our package, if its a mandatory change, let me know I will change it back

aamitn committed:
Update Bitmutex.Winhider.installer.yaml

aamitn committed:
Update Bitmutex.Winhider.installer.yaml
